examples/complete/test.tfvars

```diff
@@ -0,0 +1,91 @@
+project_name = "codebuild-osahon"
+
+source_type = "GITHUB"
+
+source_location = "https://github.com/debasish-sahoo-nttd/sample-dotnetcore-app.git"

```

Comment:
Let's make a note to replace this with a public repo.

Reply:
I could replace it.But this is a public repo

Reply:
I made https://dev.azure.com/launch-dso/platform-accelerators/_workitems/edit/310 for this.  It's currently pointing at Debasish's account.